easybuild/framework/easyblock.py

```diff
@@ -100,6 +100,11 @@
 
 MODULE_ONLY_STEPS = [MODULE_STEP, PREPARE_STEP, READY_STEP, SANITYCHECK_STEP]
 
+# constants for library checking (TODO: should probably be configurable)
+LIB_DIRS = ['lib', 'lib64']
+LIB_STATIC_SUFFIXES = ['a']
+LIB_SHARED_SUFFIXES = [get_shared_lib_ext()]
```

Comment:
Shouldn't one suffix suffice? I don't see it happening soon that we'll have multiple suffixes?

It will also simplify things below a bit.